lib/tasks/default.rake

```diff
@@ -373,6 +373,26 @@ task :save_en do
   sh 'cp -p config/locales/en.yml config/locales/en.yml.ORIG'
 end
 
+desc 'Test locale translation'
+task :test_locale do
+  # Use 'system' so we return the error code (error=0 is okay)
+  system 'rails test test/models/translations_test.rb'
+end
+
+desc 'Fix locale text'
+task :fix_locale do
+  # Google Translate generates text that has predictable errors; here
+  # are ways to fix them automatically.  Commented out, instead of being
+  # removed entirely, so that we can quickly restore them in a pinch.
+  sh %q{ruby -pi -e "sub(/< a /, '<a ')" } + files
+  sh %q{ruby -pi -e "sub(/< \057/, '</')" } + files
+  sh %q{ruby -pi -e "sub(/<\057 /, '</')" } + files
+  sh %q{ruby -pi -e "sub(/<Strong>/, '<strong>')" } + files
+  sh %q{ruby -pi -e "sub(/<Em>/, '<em>')" } + files
+  sh %q{ruby -pi -e "sub(/href = /, 'href=')" } + files
+  Rake::Task['test_locale'].invoke
```

Comment:
That's true. It probably doesn't matter, because these run really quickly, but I should fix that anyway.

My current plan is to automatically run the fix script if the first test after sync fails. That way, we know exactly what went wrong but can temporarily fix it in many cases. That way we won't slow down delivery of translations to users while we fix the original data.